mlflow/sklearn/__init__.py

```diff
@@ -937,8 +963,9 @@ def autolog(
 
         **Limitations**
            - MLflow can only map the original prediction result object returned by a model
-             prediction API (including predict / predict_proba / transform, but excluding
-             fit_predict / fit_transform.) to an MLflow run. MLflow cannot find run information
```

Comment:
Why are we making this change?

Reply:
I add patch support for an additional method `predict_log_proba`